src/main/java/com/exasol/dbbuilder/dialects/postgres/PostgreSqlImmediateDatabaseObjectWriter.java

```diff
@@ -0,0 +1,41 @@
+package com.exasol.dbbuilder.dialects.postgres;
+
+import java.sql.Connection;
+
+import com.exasol.dbbuilder.dialects.*;
+
+/**
+ * PostgreSQL {@link DatabaseObjectWriter}.
+ */
+public class PostgreSqlImmediateDatabaseObjectWriter extends AbstractImmediateDatabaseObjectWriter {
+
+    /**
+     * Create a new instance of {@link PostgreSqlImmediateDatabaseObjectWriter}.
+     * 
+     * @param connectionToPostgres connection to the PostgreSQL database.
+     */
+    public PostgreSqlImmediateDatabaseObjectWriter(final Connection connectionToPostgres) {
+        super(connectionToPostgres);
+    }
+
+    @Override
+    protected String getQuotedColumnName(final String columnName) {
+        return PostgreSqlIdentifier.of(columnName).quote();
+    }
+
+    @Override
+    public void write(final User user) {
+        writeToObject(user, "CREATE USER " + user.getFullyQualifiedName() + " PASSWORD '" + user.getPassword() + "'");
+    }
+
+    @Override
+    public void write(final User user, final GlobalPrivilege... privileges) {
+        throw new UnsupportedOperationException(
```

Comment:
Should we use the error-reporting-message-builder, or you are planning to do it in the future?

Reply:
not sure if it's worce adding int right now just for that single message. I'll open a ticket to add it to the whole project. Ok?